linux_os/guide/system/software/integrity/fips/grub2_enable_fips_mode/bash/shared.sh

```diff
@@ -26,8 +31,13 @@ if grep -q '^GRUB_CMDLINE_LINUX=".*boot=.*"'  /etc/default/grub; then
 	sed -i 's/\(^GRUB_CMDLINE_LINUX=".*\)boot=[^[:space:]]*\(.*"\)/\1 boot=UUID='"${BOOT_UUID} \2/" /etc/default/ grub
 else
 	# no existing boot=arg is present, append it
-	sed -i 's/\(^GRUB_CMDLINE_LINUX=".*\)"/\1 boot=UUID='${BOOT_UUID}'"/'  /etc/default/grub
+	sed -i "s/\(^GRUB_CMDLINE_LINUX=\".*\)\"/\1 boot=UUID=${BOOT_UUID}\"/" /etc/default/grub
```

Comment:
This line could be the fix for #3136!

Reply:
I tested this line that currently exists in the code, and it did seem to work properly even if it was imperfect.

Your referenced issue looks like it was fixed with this commit:
https://github.com/ComplianceAsCode/content/commit/2a044d57f9200a281c105453b263b3efdee2a500

Anyone with that issue was most likely using an older version that did not contain that change.